pkg/client/statistics.go

```diff
@@ -0,0 +1,30 @@
+package client
+
+import (
+	"time"
+)
+
+type QueueStats struct {
```

Comment:
A comment on what each of those attributes represents would be nice.

Reply:
Fixed! I should have marked it as draft. I have just pushed it before I went home.